app/views/shared/_header.html.erb

```diff
@@ -5,34 +5,48 @@
 
     <ul class="nav navbar-nav">
       <% if current_user %>
-          <li class="dropdown <%= 'active' if request.path.starts_with?('/projects') %>">
-            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Projects <b class="caret"></b></a>
+        <li class="dropdown <%= 'active' if request.path.starts_with?('/projects') %>">
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown">Projects <b class="caret"></b></a>
+          <ul class="dropdown-menu">
+            <% current_user.starred_projects.alphabetical.each do |project| %>
+                <li>
+                  <%= link_to project_path(project) do %>
+                      <span class="glyphicon glyphicon-star"></span> <%= project.name %>
+                  <% end %>
+                </li>
+            <% end %>
+            <% (Project.alphabetical.to_a - current_user.starred_projects).each do |project| %>
+                <li>
+                  <%= link_to project_path(project) do %>
+                      <span class="glyphicon">&nbsp;</span> <%= project.name %>
+                  <% end %>
+                </li>
+            <% end %>
+          </ul>
+        </li>
+        <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+          <li class="dropdown <%= 'active' if request.path.starts_with?('/dashboards') %>">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown">Environments <b class="caret"></b></a>
             <ul class="dropdown-menu">
-              <% current_user.starred_projects.alphabetical.each do |project| %>
-                  <li>
-                    <%= link_to project_path(project) do %>
-                        <span class="glyphicon glyphicon-star"></span> <%= project.name %>
-                    <% end %>
-                  </li>
-              <% end %>
-              <% (Project.alphabetical.to_a - current_user.starred_projects).each do |project| %>
-                  <li>
-                    <%= link_to project_path(project) do %>
-                        <span class="glyphicon">&nbsp;</span> <%= project.name %>
-                    <% end %>
-                  </li>
+              <% Environment.all.each do |env| %>
+                <li>
+                  <%= link_to dashboard_path(env) do %>
+                    <span class="glyphicon">&nbsp;</span> <%= env.name %>
```

Comment:
this `class="glyphicon"` thing is reused twice, maybe put it into a descriptive helper ?